EXT_BeamPhysics.md

```diff
@@ -54,11 +54,11 @@ The following records are defined for the file root group.
 
 - `latticeName`
   - type: Optional *(string)*
-  - description: The name of the lattice.
+  - description: The name of the lattice. This name is a descriptive string. Some programs allow a descriptive string to be associated with the lattice.
 
 - `latticeFile`
   - type: Optional *(string)*
-  - description: The location of the root lattice file.
+  - description: The  name of the root lattice file. This name may contain a path component.
```

Comment:
This is a no-issue, just spotted a superfluous space:
```suggestion
  - description: The name of the root lattice file. This name may contain a path component.
```

Reply:
Fixed